pkg/osbuildmonitor/monitor.go

```diff
@@ -0,0 +1,138 @@
+package osbuildmonitor
+
+import (
+	"bufio"
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+)
+
+// Status is a high level aggregation of the low-level osbuild monitor
+// messages. It is more structured and meant to be used by UI frontends.
+//
+// this is intentionally minimal at the beginning until we figure
+// out the best API, exposing the jsonseq direct feels too messy
+// and based on what we learn here we may consider tweaking
+// the osubild progress
+
+type Status struct {
```

Comment:
The empty line prevents godoc from connecting the docstring with the type, so it doesn't show up in the rendered documentation.